tests/CollectionTest.php

```diff
@@ -553,6 +602,57 @@ public static function eachDataProvider(): array
         ];
     }
 
+    #[DataProvider('chunkEachDataProvider')]
+    public function testChunkEach(Collection $collectionToChunk, int $chunkSize, array $expectedChunks): void
+    {
+        $expectedLambda = function(CollectionStub $collection) use ($expectedChunks): void {
+            static $i = 0;
```

Comment:
I am not a fan of static variables. Also this is not the "expected" lambda, This is the lambda (anonymous) function.

So please redo this to not depend on static vars on the anonymous function and instead pass it by reference

``` php
        $i = 0;
        $collection->eachChunk(
            $chunkSize,
            function(CollectionStub $collection) use (&$i, $expectedChunks): void {
                self::assertEquals($collection, $expectedChunks[$i++]);
            }
        );
```

Reply:
Will refactor.